x/nft/go.mod

```diff
@@ -1,6 +1,8 @@
 module cosmossdk.io/x/nft
 
-go 1.21
+go 1.21.0
+
+toolchain go1.21.1
```

Comment:
Why adding toolchain constraint here?

Reply:
go.mod automatically added it, let me manually revert it.